### ISSUE TITLE
PointFileInfo<E57FileHeader> not closing file stream

### ISSUE THREAD

**Issue by rickrain** [aardvark-platform/aardvark.algodat#22]
This method is opening a `FileStream` [here](https://github.com/aardvark-platform/aardvark.algodat/blob/c3d9a7ecb2bf38cb396515fec94ba1655c3eeb81/src/Aardvark.Data.E57/ImportE57.cs#L278) but it doesn't close it.

We're able to work around this by forcing a `System.GC.Collect()` call, but would prefer not to have to do that.

**Comment by stefanmaierhofer** [comment]
Hi @rickrain 
You are right, this seems a little bit strange.
Would you be so kind to provide a link to your code base or repository where this is used, so I can make sure to fully understand the context before making any changes.
Thanks, Stefan

**Comment by rickrain** [comment]
Hi @stefanmaierhofer, I don't have a repo to point you to. Was just experimenting with something and ran into this. For additional context, I used [PointFileInfo](https://github.com/aardvark-platform/aardvark.algodat/blob/c3d9a7ecb2bf38cb396515fec94ba1655c3eeb81/src/Aardvark.Data.E57/ImportE57.cs#L275) and then later tried to delete the file. But, since the file handle is left open the delete fails.

I just submitted PR #23 for review.
